ansible_base/lib/redis/client.py

```diff
@@ -106,12 +106,12 @@ def _get_hosts(self) -> None:
 
         translated_generic_exception = ImproperlyConfigured(_('Unable to parse cluster_hosts, see logs for more details'))
```

Comment:
cluster_hosts -> redis_hosts in exception text?

Reply:
Great catch, fixed!